### DISCUSSION TITLE
Add image to system prompt

### DISCUSSION THREAD

**Discussion by davide445** [FlowiseAI/Flowise#2091] [Q&A] [answered]
Asking GPT4/Claude3 to answer to RAG data from a PDF, but in this PDF there are a complex table.
 
The best way to have the table data in the LLM is to copy the table as image, due even unstructured.io seems not able to reliably extract data from complex tables.

I want this table image to be passed as part of the system prompt so appear to the user as part of the knowledge base the LLM in answering on.

There is a way to achieve this, or maybe there is a better way and this is not the right path.

**Comment by llattarini1982** [comment]
Hello @davide445  did u find the answer? I have the same issue

**Reply by davide445** [reply, marked as answer]
Hi @llattarini1982 in the end I placed the image in postimages.org (I suppose any equivalent will work) and referenced the direct link in the prompt
